src/index.js

```diff
@@ -1,27 +1,38 @@
-const API_URL =  "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores";
-
-const getData =  ()=>{
-  const response = fetch(API_URL, config);
-  let data = response.json(display);
-  display(data);
-  console.log(data);
-  console.log(data.result[0]);
-
-};
-
-getData();
-
-
-const displayContainer = document.querySelector('.table_item');
-let feed = '';
-
-const display = (data) => {
-
-for(let i = 0; i <data.result.length; i++) {
-feed +=  `<tr><td>${data.result[i].user}</td><td>${data.result[i].score}</td></tr>`
-
+import './style.css';
+import './modules/submitScore';
+
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores';
+
+function getScores() {
+  const displayContent = document.getElementById('table_item');
+  const scoreList = document.createElement('li');
+  const config = {
+    headers: {
+      'Access-Control-Allow-Origin': origin,
+      Accept: 'application/json',
+    },
+  };
+  fetch(API_URL, config)
+    .then((response) => response.json())
+    .then((data) => {
```

Comment:
- [ ] Kindly use the `async and await` function as specified in the project requirements. You could do something like this to get the data from the **_API_**

```
econst fetchFromApi = async () => {
  const response = await fetch(API_URL);
  const data = await response.json();
  // then write logic for use of data fetched;
}
```

- [ ] It would be nice to use _**ES6**_ syntax for initializing this project.

Reply:
Resolved, thanks for your advice.